components/movementsensor/gpsnmea/gpsnmea.go

```diff
@@ -110,7 +110,7 @@ func init() {
 
 const (
 	connectionType = "connection_type"
-	i2cStr         = "I2C"
```

Comment:
sigh, this is a breaking change unfortunately and will need a migration. someone changed it back from lower case and I'm quite annoyed.

silly, but until we can break it could you add a helper function to figure out if the config input is "I2C" and convert it? And make a follow-up ticket, we will break configs anyway with this scope so we'll do all of them together.

Reply:
will do!